dev/inc/RuntimeClassHelpers.h

```diff
@@ -229,7 +216,7 @@ struct ReferenceTracker : public ImplT<D, I ..., ::IReferenceTrackerExtension>,
 
         if (!queued)
         {
-            DeleteInstance(instance);
+            self.reset();
         }
     }
     catch (...) {}
```

Comment:
Intentionally discarding exception info? You can use winrt::terminate() if you wish to originate with full debug info.

Reply:
I don't know why this was written this way. I changed the function to noexcept.